### ISSUE TITLE
Lines with just braces, commas, etc. are marked as not covered

### ISSUE THREAD

**Issue by porglezomp** [xd009642/tarpaulin#16]
I'm seeing a lot of closing curly braces, parenthesis, commas, etc. being marked as not covered lines, when they should probably be uninteresting lines instead.
For an example, see: https://coveralls.io/builds/12605055/source?filename=src%2Fparser.rs

(Thanks for the great tool, this was the nicest setup I've ever had with getting code coverage!)

**Comment by xd009642** [comment]
I'm aware of this issue. Still mulling it over how I want to deal with it given they're down as lines cause the line is tied to an actual instruction (although could be an end to a block and therefore a not really an executable instruction).

Thanks for the compliment, finding kcov a pain to use is why I did this :)

**Comment by porglezomp** [comment]
I've got another case of this here where some `assert_eq!(` in the tests are being marked as not-covered: https://coveralls.io/builds/12605055/source?filename=src%2Fparse_util.rs#L330

**Comment by xd009642** [comment]
Hi @porglezomp, I've sorted the first part of your issue in a6cdd73e237851dc2b58bf40c6991ac9fb7219bc (well I just realised I missed out the comma but that'll be in the commit afterwards).

Once my testing is done the syntax analysis feature branch will be merged and I'll close this issue and release a new version of tarpaulin. I think the assert_eq! should have been sorted by some other changes between July and now but if not or if you see any other anomalies feel free to let me know :smile:
